gdc_client/version.py

```diff
@@ -1 +1 @@
-__version__ = 'v1.3.0'
+__version__ = 'v1.3.1'
```

Comment:
This is probably ok for now but we don't normally bump version numbers until we're ready to do a release.

Reply:
Ah ok. This is something that Ian asked me to do for his build. Will put it back.